devops/openshift/redis-ha-deploy.yml

```diff
@@ -0,0 +1,216 @@
+                      name: ${NAME}-creds
+              lifecycle:
+                preStop:
+                  exec:
+                    command:
+                      - /bin/sh
+                      - -c
+                      - redis-cli -h $HOSTNAME shutdown save
+              livenessProbe:
+                exec:
+                  command:
+                    - /bin/sh
+                    - -c
+                    - test "$(redis-cli -h $HOSTNAME ping)" == "PONG"
+                initialDelaySeconds: 10
+                timeoutSeconds: 1
+                failureThreshold: 3
+              ports:
+                - containerPort: ${{REDIS_PORT}}
+                  name: client
+                - containerPort: ${{REDIS_BUS_PORT}}
+                  name: gossip
+              readinessProbe:
+                exec:
+                  command:
+                    - /bin/sh
+                    - -c
+                    - test "$(redis-cli -h $HOSTNAME ping)" == "PONG"
+                initialDelaySeconds: 15
+                timeoutSeconds: 1
+                failureThreshold: 3
+              resources:
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+              volumeMounts:
+                - name: conf
+                  mountPath: /conf
+                  readOnly: false
+                - name: data
+                  mountPath: /data
+                  readOnly: false
+                - name: script
+                  mountPath: /script
+                  readOnly: false
+          volumes:
+            - name: conf
+              secret:
+                secretName: ${NAME}-creds
+            - name: script
+              configMap:
+                name: ${NAME}
+                defaultMode: 0755
+      volumeClaimTemplates:
+        - metadata:
+            name: data
+            labels:
+              app: ${NAME}
+          spec:
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: ${PVC_SIZE}
+          storageClassName: ${STORAGE_CLASS}
+parameters:
+  - name: NAME
+    description: The name of the application for labelling all artifacts.
+    displayName: Application Name
+    required: true
+    value: redis
+  - name: REPLICAS
+    description: The number of StatefulSet replicas to use.
+    displayName: Replicas
+    required: true
+    value: "6"
+  - name: IMAGE_REGISTRY
+    description: The base docker registry
+    displayName: Docker Image Registry
+    required: true
+    value: docker.io
+  - name: IMAGE_NAME
+    description: Redis Image Name
+    displayName: Image Name
+    required: true
+    value: redis
+  - name: IMAGE_TAG
+    description: Redis Image Tag
+    displayName: Image Tag
+    required: true
+    value: 6.0.9-alpine
```

Comment:
We are passing the IMAGE_TAG in the makefile already, so the value here will be a duplication.